test/metabase/query_processor/middleware/binning_test.clj

```diff
@@ -177,46 +174,6 @@
             (is (= [[-30.00M -60.00M 1]]
                    (mt/rows (qp/process-query query))))))))))
 
-(deftest ^:parallel fuzzy-metadata-matching-test
```

Comment:
The test does not pass after disabling broken card refs. Most likely the underlying issue is not fixed.